src/gpb_compile.erl

```diff
@@ -8489,7 +8506,7 @@ type_default(Type, Defs, Opts, GetTypeDefault) ->
     if Type == string ->
             case get_strings_as_binaries_by_opts(Opts) of
                 true ->
-                    list_to_binary(GetTypeDefault(Type, Defs));
+                    iolist_to_binary(GetTypeDefault(Type, Defs));
```

Comment:
No idea.  I may have noticed that it's the only usage of `list_to_binary` in the file:
```
$ git grep -c iolist_to_binary src/gpb_compile.erl
src/gpb_compile.erl:9
$ git grep -c list_to_binary src/gpb_compile.erl
src/gpb_compile.erl:10
```
but otherwise have no recollection.  I'll revert.